### PR TITLE
Introduce CSS lint

### PR DESCRIPTION
**Pull request by axic**
Running csslint currently show 271 errors/warnings.

### CODE REVIEW

**Comment by axic** [comment]
@Denton-L since you have looked at CSS before, are you interested in looking at the warnings reported and see if it makes sense fixing any of them?

**Comment by axic** [comment]
Disabled the rule for alphabetical order. Maybe it is useful?

We could disable other rules too if needed.

**Comment by Denton-L** [comment]
I'm not very well-versed at CSS but I'll give it a look.

**Comment by Denton-L** [comment]
This is outside the scope of my knowledge as I'm not really a web developer... I'll have to hand this off to someone else.

**Comment by axic** [comment]
@chriseth @Denton-L can you test and review this? It only fails on serious errors but reports the warnings also.

**Comment by Denton-L** [comment]
In my browser, (Chromium 52.0.2743.116 on Linux), the size of the contract seems to be displayed strangely. See below.

2016-08-31-143147_442x401_scrot

**Comment by axic** [comment]
Have look at the last two changes. Im not sure the first or the second of the duplicate entries take precedence.

File: assets/css/universal-dapp.css [hunk 1]
```diff
@@ -57,7 +57,6 @@
 	display: inline-block;
 	padding: 0.2em;
 	background-color: rgba( 255,255,255,0.5 );
-	display: block;
```
Comment (chriseth) [inline assets/css/universal-dapp.css]: The other one should be removed.

Comment (axic) [inline assets/css/universal-dapp.css]: Fixed.

**Pull request merged**